evadb/functions/function_bootstrap_queries.py

```diff
@@ -202,6 +202,18 @@
             MODEL 'yolov8n.pt';
         """
 
+stablediffusion_function_query = """CREATE FUNCTION IF NOT EXISTS StableDiffusion
+        IMPL '{}/functions/stable_diffusion.py';
+        """.format(
+    EvaDB_INSTALLATION_DIR
+)
+
+dalle_function_query = """CREATE FUNCTION IF NOT EXISTS StableDiffusion
```

Comment:
These two parameters are not used.

Reply:
I did not add it to the list of queries to be loaded by default everytime, so technically yeah, I can remove these safely. I just added it in case we need it later.

Reply:
Let's add them to the default every time. What do you think? Those two functions's setup does not do much. So there won't be a lot of overheads.

Reply:
Makes sense. I'll add it then.